### PR TITLE
Carina for Windows Developers blog post

### PR DESCRIPTION
**Pull request by DonSchenck**

### CODE REVIEW

File: _posts/2015-11-19-carina-windows-developers.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+## Carina in four steps
+There are only four steps to get rolling:
+1. Create a Carina cluster.
+2. Prepare you local machine's environment.
+3. Get the Docker client on your local machine.
+4. Start one or more containers in your cluster.
+
+### Create a Carina cluster
+To create a cluster, simply log in to GetCarina.com (if you don't have an account, creating one is a cinch -- an email address and password and you're in), click the 'Add Cluster...'' link, give the cluster name, and click the 'Create Cluster' button. It is literally that easy.
```
Comment (stanzikratel) [inline _posts/2015-11-19-carina-windows-developers.md]: add "a" before "name": give the cluster "a" name

File: _posts/2015-11-19-carina-windows-developers.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+```
+iex (wget https://download.getcarina.com/dvm/latest/install.ps1)
+
+```
+For example:
+
+![Install dvm on Windows]({% asset_path 2015-11-19-carina-windows-developers/windows_dvm_install.png %})
+
+The following PowerShell will check to see if you have a profile and will create one if you don't have one:
```
Comment (stanzikratel) [inline _posts/2015-11-19-carina-windows-developers.md]: add "command" after PowerShell

File: _posts/2015-11-19-carina-windows-developers.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+## What Next?
+
+### Running MySQL and Wordpress
+
+Get something running ... now. Right now. The Carina documentation has instructions for running <a href="https://getcarina.com/docs/tutorials/wordpress-apache-mysql/">MySQL and Wordpress in Docker containers</a>.
+
+### ASP.NET in a container
+
+Microsoft will soon be delivery the final version of the ASP.NET Docker container, allowing you to run your ASP.NET web sites inside a Docker container, running on top of the Linux kernel. It's a brave new world. If you're anxious to get started (remember, it's a beta and may change), consider the ramifications of the following:
```
Comment (stanzikratel) [inline _posts/2015-11-19-carina-windows-developers.md]: delivery: change to "delivering"

**Comment by DonSchenck** [comment]
The typos that Constance pointed out have been fixed.

**Comment by stanzikratel** [comment]
+ 1

**Comment by kmsholcomb** [comment]
Hi @DonSchenck. The attached Word document has my suggested edits and some comments. Although I can leave comments in the file here, I always start with marking up a Word doc, and as we are short on time, I'm just giving you the doc. And just in case you have any trouble with viewing the markup, I'm also attaching a PDF of the Word doc. :) 

Please let me know if you have any questions. Thanks!

[Carina for NET developers_editsKH.pdf](https://github.com/getcarina/getcarina.com/files/38416/Carina.for.NET.developers_editsKH.pdf)
[Carina for NET developers_editsKH.docx](https://github.com/getcarina/getcarina.com/files/38417/Carina.for.NET.developers_editsKH.docx)

File: _posts/2015-11-19-carina-windows-developers.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+A container is like a VM minus the operating system/kernel. It uses the host instead of packaging and running its own, so there's much less overhead. It's like a tiny, preconfigured server that runs in its own space.
+
+## Get Carina
+
+Carina by Rackspace is the master key that allows you to quickly get up to speed with Docker, see what containers are all about, and begin *now* to expand your experience. Trends such as *The 12-Factor App* and *microservices* can be explored and implemented without wasting time with minutiae.
+
+Containers need a place to run. If you're running Docker -- a specific implementation of the containers concept -- on your local machine, you use VirtualBox as your Docker host.
+
+But with Carina, you don't need to run Virtual Box on your local machine. Instead of using your CPU cycles, Carina runs on Rackspace bare metal servers, meaning it's fast to start and run. You can use your Carina clusters (your Docker hosts in Carina) from any client: Mac, Linux, or Windows. Let's look at a Windows setup.
```
Comment (kmsholcomb) [inline _posts/2015-11-19-carina-windows-developers.md]: VirtualBox 
(one word)

**Comment by kmsholcomb** [comment]
Just one more tiny comment (see the file). :+1:

**Comment by DonSchenck** [comment]
This post is ready to be merged.

**Pull request merged**